README.md

```diff
@@ -84,7 +84,7 @@ To get a local copy up and running follow these simple example steps.
 
 ### Live Demo
 
-To see live demo, click [here](https://haywayaheadshot.github.io/Maths-Magician/)
+To see live demo, click [here](https://636193f25dc0385a868d3858--dashing-cranachan-aa1109.netlify.app/)
```

Comment:
- [ ] You have done a good job with your README, but please try to use this [template](https://github.com/microverseinc/readme-template) as a guide and model your README after it because there are some sections of your README that default from React and needs to be removed. 🙏

Reply:
@codepantha Thank you for reviewing my code